### ISSUE TITLE
Holdings Per Widget

### ISSUE THREAD

**Issue by seniorm0ment** [premnirmal/StockTicker#321]
Was trying to setup two different widgets with two different holding amounts.
For example, widget one would show my held shares in my brokerage account.
Widget 2 would show the held shares in my roth ira.

However, if the same ticker is in both say KO for example.
It merges all holdings. So I can't track KO on JUST my roth ira on one widget, and my holdings on my brokerage on another. 
If that makes sense. 

Basically what I am asking for is to be able to have two isolated portfolios. One to track my main stuff, another to track just my roth ira.
